staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/cel_validation.go

```diff
@@ -25,6 +25,7 @@ import (
 	structuralschema "k8s.io/apiextensions-apiserver/pkg/apiserver/schema"
 	"k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/model"
 	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/apiserver/pkg/cel"
```

Comment:
A couple other potential destinations for the code:

`k8s.io/apiserver/pkg/util/cel`
`k8s.io/apimachinery/pkg/util/cel`

I can see arguments for both. I have a slight preference for `apimachinery` since our "Kubernetes CEL library code" has very few dependencies, and could be useful outside of apiservers, e.g. in clients or in CI/CD pipelines..

Curious what others think.

cc @deads2k @lavalamp @liggitt

Reply:
cel and antlr are on the heavy side for apimachinery (means it becomes a dep of client-go and api as well)

Reply:
Good point. I keep forgetting that ANTLR gets pulled in.